core/device_factory.h

```diff
@@ -29,18 +29,20 @@ OIDN_NAMESPACE_BEGIN
     virtual Ref<Device> newDevice(const int* deviceIDs, const cudaStream_t* streams, int numPairs) = 0;
   };
 
+class MetalDeviceFactoryBase : public DeviceFactory
+{
+public:
+  using DeviceFactory::newDevice;
+  virtual Ref<Device> newDevice(const MTLCommandQueue_id* commandQueues, int numQueues) = 0;
+};
+
```

Comment:
Why was this added?

Reply:
Not sure why git diffed this, I reverted the file